src/modules/tas_studio/mod.rs

```diff
@@ -1470,6 +1521,7 @@ fn add_frame_bulk_hud_lines(text: &mut Vec<u8>, bulk: &FrameBulk, last_frame: Op
         write!(text, "  X Pos: {:.1}\0", last_frame.state.player.pos.x).unwrap();
         write!(text, "  Y Pos: {:.1}\0", last_frame.state.player.pos.y).unwrap();
         write!(text, "  Z Pos: {:.1}\0", last_frame.state.player.pos.z).unwrap();
+        write!(text, "  Stamina: {:.1}\0", last_frame.state.player.stamina_time).unwrap();
```

Comment:
Extract into separate PR, or honestly I'll just do this myself today

Reply:
Go ahead with this one.